src/main.rs

```diff
@@ -25,9 +26,11 @@ pub fn main() {
     Logger::init(LogContext::Volta, verbosity).expect("Only a single logger should be initialized");
 
     let mut session = Session::init();
-    session.add_event_start(ActivityKind::Volta);
+    let volta_argv = env::args().collect::<Vec<String>>().join(" ");
```

Comment:
A concern I have here is that `env::args` will [panic](https://doc.rust-lang.org/stable/std/env/fn.args.html#panics) if any of the args are not valid UTF-8. Elsewhere in the app I think we've explicitly used `args_os` to avoid that, and dealt with the potential issue of non-UTF-8 text with the `OsString` type.

Reply:
Oh great callout, I didn't see that when I was adding this. I'll change things to use `env::args_os` instead.